source/preferences.cpp

```diff
@@ -658,9 +658,10 @@ bool LoadPrefs()
 	sprintf(filepath[3], "sd:/%s", APPFOLDER);
 	sprintf(filepath[4], "usb:/%s", APPFOLDER);
 #else
-	numDevices = 2;
+	numDevices = 3;
 	sprintf(filepath[0], "carda:/%s", APPFOLDER);
 	sprintf(filepath[1], "cardb:/%s", APPFOLDER);
+	sprintf(filepath[1], "cardc:/%s", APPFOLDER);
```

Comment:
should be filepath[2]

Reply:
pretty sure this is outdated and he could pull from your master instead :)